### ISSUE TITLE
[Request] Soul sacrifice delta

### ISSUE THREAD

**Issue by molecopopo** [Electry/VitaGrafix#32]
Would a patch be possible for this game? Thanks

**Comment by Errol1886** [comment]
Same here. I would love to see a native resolution patch for this game.

**Comment by nad45** [comment]
A resolution hack is available if you want
It's a lot easier than modifying the eboot

https://www.reddit.com/r/vitahacks/comments/535dvo/so_i_tried_to_dig_around_soul_sacrifice_delta/

Until @Electry add the game in Vitagrafix, you can try

**Comment by ChiSys** [comment]
Here is a repatch mod for it 
[PCSF00532.zip](https://github.com/Electry/VitaGrafix/files/2338123/PCSF00532.zip)

**Comment by Errol1886** [comment]
@ChiSys 

Will the repatch mod you posted above make the game run in native resolution?

**Comment by ChiSys** [comment]
Yes, it's the same like in the reddit post only for repatch. Its for the 1.30 EU version. If you use an other region rename the folder PCSF00532 to the version you use.
